### DISCUSSION TITLE
"Convert to issue" button missing on Pull Requests, I see it only on Issues

### DISCUSSION THREAD

**Discussion by richardeakin** [community/community#8893] [Projects and Issues]
Title says it all - is this a known limitation?

Use case: my basic workflow during a PR is to create the PR and then keep a running list of outstanding tasks before it should be merged. If we find that we want to merge the PR before finishing all the sub-tasks, then we want to create issues out of the sub-tasks. 

Up until now, I've been doing this manually. It'd be great if I could just 'convert remaining tasks to issues", which is a feature in Jira, or at least if the (new?) feature I see in Issues:

![image](https://user-images.githubusercontent.com/5132127/146230929-abf078d2-3bb0-4a46-9b3a-ce15167868b7.png)

was also available in a Pull Request (don't see that same option when I hover the left side of a task).

**Comment by rickyn-croesus** [comment]
Just stumbled on the same issue. 

Expected the same functionality as in Issues, but "Convert to issue" is not available for tasks list in Pull Requests.
